### PR TITLE
[Remove Spring extender usages] LPS-159586 commerce-price-list-service | 8-8

### PR DESCRIPTION
**Pull request by dantewang**
https://issues.liferay.com/browse/LPS-159586

### CODE REVIEW

**Comment by dantewang** [comment]
ci:test:commerce

**Comment by dantewang** [comment]
@catarineaguiar Could you please review the test results when they finish? Thanks!

**Comment by dantewang** [comment]
ci:test:sf

**Comment by dantewang** [comment]
ci:test:relevant

**Comment by dantewang** [comment]
ci:test:commerce

**Comment by dantewang** [comment]
@zhangyan0701 Could you review the latest test results? Thanks!

**Comment by zhangyan0701** [comment]
No regressions found.

File: modules/apps/commerce/commerce-price-list-service/src/main/java/com/liferay/commerce/price/list/service/impl/CommerceTierPriceEntryLocalServiceImpl.java [hunk 1]
```diff
@@ -211,8 +211,9 @@ public CommerceTierPriceEntry addCommerceTierPriceEntry(
 
 		// Commerce price entry
 
-		_commercePriceEntryLocalService.setHasTierPrice(
-			commercePriceEntryId, true, bulkPricing);
+		CommercePriceEntryUtil.setHasTierPrice(
```
Comment (tinatian) [inline modules/apps/commerce/commerce-price-list-service/src/main/java/com/liferay/commerce/price/list/service/impl/CommerceTierPriceEntryLocalServiceImpl.java:214]: @dantewang This change is going to skip the reindex in _commercePriceEntryLocalService#setHasTierPrice. I did not see the circular dependencies here, could you please add the details to the commit message?

Comment (dantewang) [reply modules/apps/commerce/commerce-price-list-service/src/main/java/com/liferay/commerce/price/list/service/impl/CommerceTierPriceEntryLocalServiceImpl.java:214]: @tinatian As far as I remember, a delete logic in `CommercePriceEntryLocalService` uses the delete method on `CommerceTierPriceEntryLocalService`. I'll double check.

**Comment by dantewang** [comment]
See https://github.com/liferay-core-infra/liferay-portal/pull/1324

**Pull request closed without merging**